pkg/flow/internal/controller/node_config_import.go

```diff
@@ -11,6 +11,7 @@ import (
 	"time"
 
 	"go.uber.org/atomic"
+	"golang.org/x/exp/maps"
```

Comment:
nit: maps is available in stdlib now: https://pkg.go.dev/maps, that should be used when possible